src/nyc_trees/js/src/progressPage.js

```diff
@@ -24,28 +24,34 @@ var progressMap = mapModule.create({
         modeDropdown: '.dropdown',
         modeButton: '.btn',
         modeChoice: '.dropdown li a',
-        legendEntries: '.entry',
+        legendEntries: '[data-mode]',
         actionBar: '#action-bar'
     },
     $actionBar = $(dom.actionBar);
 
 $(dom.modeChoice).click(onModeChanged);
 
-$(dom.modeChoice).first().trigger('click');
+loadLayers($(dom.modeChoice).first());
```

Comment:
This change breaks filtering of legend entries on first load. In this picture there should only be 2 legend entries, not 4.

![image](https://cloud.githubusercontent.com/assets/2162993/7182262/27231e86-e41e-11e4-8e5f-5d81d46983b6.png)